src/go/rpk/pkg/cli/cluster/txn/abort.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package txn
+
+import (
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/kadm"
+)
+
+func newAbortCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var (
+		topic       string
+		partition   int32
+		startOffset int64
+	)
+	cmd := &cobra.Command{
+		Use:   "abort",
+		Short: "Abort an open transaction for a given partition",
+		Long: `Abort an open transaction for a given partition.
+
+This command aborts a single partition in a transaction, allowing the
+last-stable-offset to advance for the partition. To be extra cautious, this
+command only aborts one partition at a time. You can run this repeatedly to
+abort all partitions that may be hanging in a single transaction.
+
+You can find potentially hanging transactions with the find-hanging command.
```

Comment:
Is there already an abort-all-partition command? It seems like that'd be pretty useful to have

Reply:
That's a great call out, but in DMing with Denis, even more fortunately for us, we can outright delete the `abort` command in this PR. Redpanda doesn't have the same flaw that Kafka proper has, so Redpanda makes it a bit more impossible to have hanging bugged transactions.

Reply:
Dropped from this PR